salt/metalk8s/repo/redhat.sls

```diff
@@ -13,9 +13,15 @@ Install yum-utils:
   pkg.installed:
     - name: yum-utils
 
-Install yum-plugin-versionlock:
+{%- if grains.osmajorrelease < 8 %}
+Install versionlock plugin:
   pkg.installed:
     - name: yum-plugin-versionlock
```

Comment:
Why don't we do this through the package name per OS map thing we already have in place?

Reply:
We totally could, I'll change it.